File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int n=s.size()-1;
        string ans="";
        while(n>=0){
            if(s[n]==' ') {
                if(ans.size()>0) break;              
               }
            else ans.push_back(s[n]);
            n--;
        }
        return ans.size();
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        s = s.strip()
        last_space_index = s.rfind(' ')
        if last_space_index == -1:
            return len(s)

        return len(s) - last_space_index - 1
        
        